packages/text/src/TextMetrics.ts

```diff
@@ -502,11 +502,16 @@ export class TextMetrics
     /**
      * Determines if char is a breaking whitespace.
      *
+     * It allows one to determine whether char should be a breaking whitespace
+     * For example certain characters in CJK langs or numbers.
+     * It must return a boolean.
+     *
      * @private
```

Comment:
this private tag will also need to be remove

Reply:
Fixed

Reply:
I am sorry that I forgot about this, and thanks @bigtimebuddy for fixing it 🙏